app/views/home/failure.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="py1 mb3 center">
+  <h2>Authentication from Login.gov was not successful! :(</h2>
```

Comment:
The failure message is already being displayed here via the flash message in the controller.